frontend/src/components/AdminQuizComp/QuizModal.tsx

```diff
@@ -61,12 +61,17 @@ const QuizModal: FC<Props> = ({ editQuizListData, SaveLoading, editQuizAdminModa
         <Modal
             isOpen={editQuizListData ? true : false}
             title={
-                <span>
-                    <PlusCircleIcon className="w-5 h-5" />&nbsp;Add Quiz
-                </span>
+                input.id && input.id ?
+                    <span>
+                        <PlusCircleIcon className="w-5 h-5" />&nbsp;Update Quiz
+                    </span>
+                    :
+                    <span>
+                        <PlusCircleIcon className="w-5 h-5" />&nbsp;Add Quiz
+                    </span>
```

Comment:
since it has the same design, use the condition only to the strings
```suggestion
                    <span>
                        <PlusCircleIcon className="w-5 h-5" />input.id ? Update Quiz : Add Quiz
                    </span>
```